examples/index.jsx

```diff
@@ -8,6 +8,7 @@ import { Route, hashHistory } from 'react-router'
 import { MobileRouter } from 'cozy-authentication'
 import 'cozy-ui/transpiled/react/stylesheet.css'
 import appIcon from './icon.png'
+import { Sprite as IconSprite } from 'cozy-ui/transpiled/react/Icon'
```

Comment:
Not sure about this one. We are now importing the full UI's sprite for 5 icons. 

What was the bundle size before and after this change?

Reply:
https://github.com/cozy/cozy-ui/pull/1626 As soon as this is merged, we will be able to import only specific icons, and the bar will not rely on the IconSprite.